Orange/base.py

```diff
@@ -99,10 +95,8 @@ def predict(self, X):
             return self.predict_storage(table)
 
     def predict_storage(self, data):
-        if isinstance(data, Storage):
+        if isinstance(data, (Table, TableSeries)):
```

Comment:
Currently, base learner works with dense and sparse data. IIUC, this would only make it work for dense data?